util/coverage/coverage.css

```
@@ -25,6 +25,15 @@ body > div {
   align-items: end;
 }
 
+#description-emoji {
+  float: left;
+  margin-right: 5px;
+}
+
+#description-text {
+  display: inline-block;
+}
```

```suggestion
#description {
  white-space: nowrap;
}

#description-emoji {
  float: left;
  margin-right: 5px;
}

#description-text {
  display: inline-block;
  white-space; normal;
  max-width: 800px;
}
```

What about something like this?

---

Ah, max-width! I didn't have that in mind. I was worrying about fixed width of description div. But Others have dynamic width.

---

And the white-space attribute, good and thanks. I learned how to handle this div going upside problem.